### DISCUSSION TITLE
GSOC '23 Applicant - Add OpenVINO support to John Snow Labs Spark NLP

### DISCUSSION THREAD

**Discussion by muskan-k** [openvinotoolkit/openvino#15949] [Google Summer of Code]
Hello OpenVINO community! 

I'm Muskan Kothari, CSE undergrad from PES University. I'm interested in contributing to OpenVINO and my previous projects in ML, NLP and Big Data put me in a comfortable position to begin my open source journey. I have gone through the pre-requisite task, but I'd also like to know and work on any open issues to get started and know more about the community. 

I'm interested in the project:` Add OpenVINO support to John Snow Labs Spark NLP ` whose mentors are `Junwen Wu, Ravi Panchumarthy`. 

How can I get in touch with them and discuss further? 

Looking forward to learning more!

Thanks,
Muskan

**Comment by adrianboguszewski** [comment]
Hi Muskan, you can ask your questions here :) I'm sure the mentors (@junwenwu @ravi9) will answer them.
